Chell0/machtbewwust
# Add Banner Section

## Description
This pull request adds a banner section to the project.

### Changes
- Implemented a banner component
- Styled the banner using Tailwind CSS

## Screenshots
![Navbar Screenshot](banner-section.png)

## How to Test
1. Pull this branch.
2. Run the development server using `bun dev`.
3. Verify the banner displays correctly and is responsive on different screen sizes.
